tests/test_arthur.py

```diff
@@ -323,6 +323,70 @@ def test_add_duplicated_task(self):
         with self.assertRaises(AlreadyExistsError):
             app.add_task("arthur.task", "backend", "category", {"a": "a", "b": "b"})
 
+    def test_add_task_no_task_id(self):
+        """Check whether an exception is thrown when the task id is missing"""
+
+        task_id = None
+        backend = "backend"
+        category = "mock_category"
+        backend_params = {"a": "a", "b": "b"}
+
+        app = Arthur(self.conn, async_mode=False)
+
+        with self.assertRaises(ValueError):
```

Comment:
I forgot to mention that these assertions should be checked with `assertRaisesRegex`, like in https://github.com/chaoss/grimoirelab-kingarthur/blob/master/tests/test_tasks.py#L299.
Otherwise, you would be hiding other possible errors because `ValueError` is raised when an error is found in any of the parameters.